src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/VisualStudioLogging.cs

```diff
@@ -220,4 +227,44 @@ private static void WaitForFileExists(string file)
             Thread.Sleep(100);
         }
     }
+
+    private static void WaitForFileFinishedWriting(string file)
```

Comment:
Yessss so glad to have the LogHub logs uncorrupted! 👏 Is there a validation run that confirms this works in CI? A while ago I think I tried something similar with `File.Open` and was still seeing the corrupted logs, but my overall implementation was a bit different which could have affected things

Reply:
This run is a failure from this branch which should prove it works: https://dev.azure.com/dnceng-public/public/_build/results?buildId=391392&view=artifacts&pathAsName=false&type=publishedArtifacts